poetry/utils/env.py

```diff
@@ -367,11 +367,19 @@ def list(self, name=None):  # type: (Optional[str]) -> List[VirtualEnv]
         else:
             venv_path = Path(venv_path)
 
-        return [
+        env_list = [
             VirtualEnv(Path(p))
             for p in sorted(venv_path.glob("{}-py*".format(venv_name)))
         ]
 
+        current_env = self.get()
+        if (
+            self._poetry.config.get("virtualenvs.in-project")
+            and current_env not in env_list
```

Comment:
Thanks a lot again :+1: 

One more thing: The current check adds the the current activated env to the list. But what we want, is to add `.venv` if it exists in the project folder, isn't it? The current path to the project folder should be available with `self._poetry.file.path.parent` at this point.